db/controllor/controller.js

```diff
@@ -54,5 +54,15 @@ const postComment = (request, response, next) => {
     })
 };
 
+const getUsers = (request, response, next) => {
+    selectAllUsers()
+    .then((users) => {
+        response.status(200).send( users );
```

Comment:
Users should be sent on an object with a key of users and a value of your users, please!